foursight_core/react/api/yaml_utils.py

```diff
@@ -0,0 +1,57 @@
+import yaml
+
+
+def load_yaml(yaml_string: str) -> dict:
```

Comment:
I have some concern about the default here. If the Yaml is in bad form, should the user not know this?